build.gradle

```diff
@@ -11,11 +11,14 @@ buildscript {
     repositories {
         google()
         jcenter()
+        maven {
+            url 'https://dl.bintray.com/novoda-oss/snapshots'
```

Comment:
Note: unfortunately we are having some issues with `bintray-release`, as the latest version of the plugin `0.9.1`is not showing up in JCenter (the issue might be related to the migration of Bintray accounts that we are performing).
In the meantime we have decided to use a snapshot release  of the plugin instead.